device/device.go

```diff
@@ -0,0 +1,110 @@
+package device
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/device/deviceType"
+	"github.com/josephroberts/edge-node-manager/radio/bluetooth"
+	"github.com/josephroberts/edge-node-manager/radio/radioType"
+	"github.com/josephroberts/edge-node-manager/radio/wifi"
+	"github.com/josephroberts/edge-node-manager/radio/zigbee"
+)
+
+type Type struct {
```

Comment:
Also `Type` is not a good type name since the reflection package has a [type Type](https://golang.org/pkg/reflect/#Type) already and so referencing `Type` in this package will be confusing for anybody who's used it.